packages/itwin/geo-tools/src/GeoToolsUiItemsProvider.tsx

```diff
@@ -3,12 +3,19 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 import { ToolbarItemUtilities, ToolbarOrientation, ToolbarUsage } from "@itwin/appui-react";
-import type { ToolbarItem, UiItemsProvider } from "@itwin/appui-react";
-
 import { GeoToolsItemDef } from "./GeoToolsItemDef";
 
+import type { GeoToolsOptions } from "./GeoToolsItemDef";
+import type { ToolbarItem, UiItemsProvider } from "@itwin/appui-react";
+
 export class GeoToolsAddressSearchProvider implements UiItemsProvider {
   public readonly id = "GeoToolsAddressSearchProvider";
+  private _opts?: GeoToolsOptions;
+
+
+  public constructor(opts?: GeoToolsOptions) {
```

Comment:
@aruniverse If some other custom API would need to be used, a custom address provider can be passed here.

This new new provider could be created by sub-classing one of the two existing providers.